Helper/Api/Client.php

```diff
@@ -143,4 +149,36 @@ public function postFile(\SplFileInfo $file, ?string $forcedFilename = null): ar
 
         return \json_decode($response->getBody()->getContents(), true);
     }
+
+    public function createFormVerification(string $value): ?int
+    {
```

Comment:
It is an int really? what about this code 001234?

Reply:
The api is generating it like this: mt_rand(100000, 999999);
It will never start with 0, but will change it?

Reply:
Fixed d18c8b2